rules/common.smk

```diff
@@ -1,17 +1,45 @@
 import pandas as pd
+from peppy import Project, SAMPLE_NAME_COLNAME as PEP_SAMPLE_COL
 from snakemake.utils import validate
 
-report: "../report/workflow.rst"
+
+def peppy_rename(df):
+    """ Rename peppy's column for sample name identification to snakemake's. """
+    if df is None:
+        return None
+    if "sample" in df.columns and PEP_SAMPLE_COL in df.columns:
+        raise Exception("Multiple sample identifier columns present: {}".format(", ".join(["sample", PEP_SAMPLE_COL])))
+    return df.rename({PEP_SAMPLE_COL: "sample"}, axis=1)
```

Comment:
What is the name that peppy uses here?

Reply:
Is it sample_name or may it be something else? If it is sample_name, I'd rather change the access to this df in the rules to use sample_name.

Reply:
Yep, `sample_name`

Reply:
this is exactly the same issue as the units vs subsample_name and can be solved in the same way.

Reply:
@johanneskoester so is this something you will change?